app/frontend/src/components/RequestForm.vue

```diff
@@ -88,8 +88,8 @@
       </v-row>
     </v-form>
     <div class="justify-center pb-8">
-      <v-btn class="request-form-cancel-btn BC-Gov-PrimaryButton light float-left" text @click="cancel()">Cancel</v-btn>
-      <v-btn class="request-form-submit-btn BC-Gov-PrimaryButton float-right" text @click="postRegistrationForm()">Submit</v-btn>
```

Comment:
These classes were added in to allow the button to be searchable in a future unit test. The intention is to have the unit test be able to "click" on a button, and then ensure that the associated call is invoked.
Granted it isn't exactly needed either at the moment since I can't seem to get `wrapper.find('.request-form-submit-btn').trigger('click')` to actually make a click happen... not sure if it's implementation error or a bug.

Reply:
Hmmm, ok. I can put them back if we want. Searched and didn't find them referenced anywhere else in the project.
For tests setting an ID might be more appropriate than a class? Not sure.

Reply:
As per discussion, let's leave the classes removed for now. If/when we need them for selection, we'll readd them in as id tags for unit tests.